### ISSUE TITLE
Can we use this for our personal? If yes, How to use it?!

### ISSUE THREAD

**Issue by tansihmittal** [wexond/browser-base#582]

**Comment by HuyHung1408** [comment]
This is a web browser so you can use it. Go to code section, find release, click asset, find the .exe file, click it to download.

**Comment by HuyHung1408** [comment]
This is a really good browser, nice animation, but still occurs some bugs.

**Comment by xnerhu** [comment]
Yes, you can use it for personal uses. Just  remember about the license and [patents](https://github.com/wexond/browser-base/blob/master/PATENTS).

**Comment by xnerhu** [comment]
> This is a really good browser, nice animation, but still occurs some bugs.

Thanks. There are bugs, but we won't fix them any time soon because we are focused on our next web browser . More info on our website [wexond.net](https://wexond.net)

[Screenshots](https://twitter.com/wexond)

**Comment by anesuc** [comment]
> > This is a really good browser, nice animation, but still occurs some bugs.
> 
> Thanks. There are bugs, but we won't fix them any time soon because we are focused on our next web browser . More info on our website [wexond.net](https://wexond.net)
> 
> [Screenshots](https://twitter.com/wexond)

Is it going to be open source like this one?

**Comment by xnerhu** [comment]
It won't be, but some components are open source like [wexond-ui](https://github.com/wexond/wexond-ui).

**Comment by anesuc** [comment]
Alright! I am working on my own custom Linux based OS so interested in potentially porting this for it but we'll see. Also got this when  I tried getting on the waitlist and got this (after filling all the other answers):
![image](https://user-images.githubusercontent.com/17863266/133910929-78acc07b-c442-4c99-9d99-abc0d75bd2e9.png)
